src/main/java/bio/terra/app/controller/converters/SqlSortDirectionConverterDesc.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.app.controller.converters;
+
+import bio.terra.model.SqlSortDirectionDescDefault;
+import java.util.Arrays;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SqlSortDirectionConverterDesc
+    extends OpenApiEnumConverter<SqlSortDirectionDescDefault> {
+
+  @Override
+  SqlSortDirectionDescDefault fromValue(String source) {
+    return SqlSortDirectionDescDefault.fromValue(source);
```

Comment:
So this is the only remaining part part that gives me pause on this PR. When is this converter run? Is it always run for this enum type when it's passed in via a web API? If so, could it handle `null` here instead of in the `SqlSortDirection` `from()` APIs? If that worked, that would be cleaner, I think. Something like
```
      return source != null ? SqlSortDirectionDescDefault.fromValue(source) : SqlSortDirectionDescDefault.DESC;
```

Reply:
Hm, ok, I'm a little stumped - So, I tried moving this logic to the converter, but it didn't work (null was still passed through to the controller). I set breakpoints on the converter code and they were never hit. I also tried removing the converter code altogether, but that breaks the conversion from a string to the enum . So it's definitely used, I'm just not sure when/how.

Reply:
It appears that the converter is only called if a non-null value is passed into the sort direction field.

<img width="879" alt="image" src="https://github.com/DataBiosphere/jade-data-repo/assets/13254229/40fe6dc4-e000-4856-8f5d-db17f72d1696">